homeassistant/components/axis/config_flow.py

```diff
@@ -84,6 +81,10 @@ def __init__(self):
 
                 self.serial_number = device.vapix.get_param(
                     VAPIX_SERIAL_NUMBER)
+
+                if self.serial_number in configured_devices(self.hass):
```

Comment:
serial == mac ?

Reply:
Yes, should I use Mac as property on device level?